plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -1408,3 +1410,48 @@ func TestEBSVolumeCountConflicts(t *testing.T) {
 		}
 	}
 }
+
+func TestPredicatesRegistered(t *testing.T) {
+	var functionNames []string
+
+	// Files and directories which predicates may be referenced
+	targetFiles := []string{
+		"./../../algorithmprovider/defaults/defaults.go", // Default algorithm
+		"./../../factory/plugins.go",                     // Registered in init()
+		"./../../../../../pkg/",                          // kubernetes/pkg, often used by kubelet or controller
+	}
+
+	// List all golang source files under ./predicates/, excluding test files and sub-directories.
+	files, err := codeinspector.GetSourceCodeFiles(".")
+
+	if err != nil {
+		t.Errorf("unexpected error: %v when listing files in current directory", err)
+	}
+
+	// Get all public predicates in priority files.
```

Comment:
s/priority/predicate

or simply "Get all public predicates" would be enough, IMO.